### DISCUSSION TITLE
How could I separate updaters for test?

### DISCUSSION THREAD

**Discussion by huangjj27** [tauri-apps/tauri#4980] [Q&A] [answered]
I'm  trying to setup an updater for both test and production, with test's endpoints coming before production's:

```json
{
  "tauri": {
    "updater": {
      "active": true,
      "endpoints": [
        "http://server.for.test/latest.json",
        "https://server.for.production/latest.json"
      ],
    }
  }
}
```
as is shown above, `server.for.test` is HTTP only which causes errors when I compile to release mode, so I think I need to separate the config -- one for the production, and the other for the test. How do I achieve this?

**Comment by JonasKruckenberg** [comment, marked as answer]
This is a little advertised feature: You can have a base config called `tauri.conf.json` or `tauri.conf.json5` and then separate more specific settings into their own config files so in your case `production.tauri.conf.json` and `test.tauri.conf.json`.

When you invoke the tauri CLI with the specific config `tauri build --config test.tauri.conf.json` it will merge both the config file you gave and the base config file!

**Reply by huangjj27** [reply]
thanks! I must miss the `--help` of the subcommands.
